app_dart/lib/src/request_handlers/push_gold_status_to_github.dart

```diff
@@ -291,21 +310,27 @@ class PushGoldStatusToGithub extends ApiRequestHandler<Body> {
   ) async {
     final Stream<IssueComment> comments = gitHubClient.issues.listCommentsByIssue(slug, pr.number!);
     await for (IssueComment comment in comments) {
-      if (comment.body!.contains(config.flutterGoldInitialAlert(_getTriageUrl(pr.number)))) {
+      if (comment.body!.contains(config.flutterGoldInitialAlert(_getTriageUrl(slug, pr.number)))) {
         return false;
       }
     }
     return true;
   }
 }
 
-Future<Map<String, dynamic>?> _queryGraphQL(GraphQLClient client, int? prNumber) async {
+Future<Map<String, dynamic>?> _queryGraphQL(
+  GraphQLClient client,
+  RepositorySlug slug,
+  int? prNumber,
```

Comment:
Same here. I do not recall why these would ever be null.

Reply:
Probably because the `PullRequest` class has the `number` property as nullable.

Reply:
Yeah I was thinking that may be so. In that case, we should probably assert somewhere that we have gotten a PR number to work with, otherwise this all breaks.